src/main/java/commaproject/be/commaserver/service/CommaServiceImpl.java

```diff
@@ -0,0 +1,86 @@
+package commaproject.be.commaserver.service;
+
+import commaproject.be.commaserver.domain.comma.Comma;
+import commaproject.be.commaserver.repository.CommaRepository;
+import commaproject.be.commaserver.service.dto.CommaDetailResponse;
+import commaproject.be.commaserver.service.dto.CommaRequest;
+import commaproject.be.commaserver.service.dto.CommaResponse;
+import commaproject.be.commaserver.service.dto.CommentDetailResponse;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class CommaServiceImpl implements CommaService {
+
+    private final CommaRepository commaRepository;
+
+    @Override
+    public CommaDetailResponse readOne(Long commaId) {
+        // user, like, comment 도메인 개발 예정
+        Comma findComma = commaRepository.findById(commaId)
+            .orElseThrow(NoSuchElementException::new);
+        Long userId = 1L;
+        int likeCount = 1;
+        List<CommentDetailResponse> comments = new ArrayList<>();
+        return new CommaDetailResponse(
+            findComma.getId(),
+            findComma.getTitle(),
+            findComma.getContent(),
+            findComma.getUsername(),
+            userId,
+            findComma.getCreatedAt(),
+            likeCount,
+            comments
+            );
```

Comment:
p3; 로직 단위로 개행으로 굽구분 지어주는 건 어떤가요?
```suggestion
        // user, like, comment 도메인 개발 예정
        Comma findComma = commaRepository.findById(commaId)
            .orElseThrow(NoSuchElementException::new);
        Long userId = 1L;
        int likeCount = 1;
        List<CommentDetailResponse> comments = new ArrayList<>();
        
        return new CommaDetailResponse(
            findComma.getId(),
            findComma.getTitle(),
            findComma.getContent(),
            findComma.getUsername(),
            userId,
            findComma.getCreatedAt(),
            likeCount,
            comments
            );
```

Reply:
개행을 로직 단위를 구분해주는 것 좋네요!! 확실히 더 읽기 편하다고 생각됩니다 ㅎㅎ 감사합니다!